app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -337,6 +330,7 @@ public boolean onContextItemSelected(MenuItem item) {
         Feed feed = ((NavDrawerData.FeedDrawerItem) drawerItem).feed;
         if (itemId == R.id.multi_select) {
             speedDialView.setVisibility(View.VISIBLE);
+            subscriptionAddButton.setVisibility(View.GONE);
```

Comment:
Could you move this to onStartSelectMode please? Then it doesn't get forgotten if we ever change to other ways of initiating multi-select in the future